.github/renovate.json

```diff
@@ -3,5 +3,6 @@
   "extends": [
     "config:base",
     ":disableDependencyDashboard"
-  ]
+  ],
+  "prConcurrentLimit": 0
```

Comment:
Can we set this to something like 100? I'd still like a limit, even if it's a high one.

Reply:
Sure